docs/integrations/spark.md

```diff
@@ -389,7 +397,32 @@ On Hadoop 2, configure:
 </configuration>
 ```
 
-(You can also set a default output committer using mapred.output.committer.class and possibly setting mapred.mapper.new-api=false.)
+The lakeFS output committer holds Hadoop working objects on temporary
+branches and merges them back to the output branch when done.  Objects
+appear atomically: it is not possible for another task to see the output
+path with any set of objects except the entire desired output set.
```

Comment:
A suggestion: replace the order of the how and what. the current version starts with how the foc works, but as a user I would rather understand WIFM first (atomicity).

Reply:
Yup!  So I moved it up to the higher-level "using the lakeFS OC" header -- which already states up-front the advantages.